udata/api/fields.py

```diff
@@ -90,3 +91,71 @@ def pager(page_fields):
             description='The previous page URL if exists')
     }
     return pager_fields
+
+
+class MaImageField(ma_fields.Field):
```

Comment:
Maybe we want to change the `Ma*Field` pattern, but don't have perfect ideas. How about `Marsh*Field`